server/mailet/mailetcontainer-camel/src/main/java/org/apache/james/mailetcontainer/impl/LocalResources.java

```diff
@@ -100,7 +100,7 @@ boolean isLocalEmail(MailAddress mailAddress) {
     }
 
     private boolean isLocaluser(MailAddress mailAddress) throws UsersRepositoryException {
-        return localusers.contains(localusers.getUser(mailAddress));
+        return localUsers.contains(localUsers.getUser(mailAddress));
```

Comment:
Not really in the scope of this PR but `UserRepository.getUser` could be rename as `UserRepository.getUsername` it would be less confusing